### DISCUSSION TITLE
Reasons and Suggestions for Increasing the Quality Multiplier of Filecoin Networks

### DISCUSSION THREAD

**Discussion by 1475Notary** [filecoin-project/FIPs#833] [Enhancements - Technical]
Author: 1475 Notary
Type: Technical
Category: Core
2023-09-15

**Foreword**
This FIP focuses on explaining why we need to increase the quality multiplier. We will discuss the pros and cons of raising the quality multiplier and the proposal to increase the quality multiplier.

**Background**
Using the Filecoin network's huge storage capacity to store data is one of the key visions of the project. The Fil+ program adjusts the Raw Byte Power (RBP) to 10x the Quality Adjusted Power (QAP) through the quality multiplier, which means a 9x increase in the chances of obtaining block rewards as compared to a sector without data from a data cap allocation. The quality multiplier is used to subsidize storage providers participating in the Fil+ program, SPs receive and store data verified by a notary (human) can gain 10x QAP, in return for which the storage providers will provide data owners with stable, long-term, competitively priced storage services.

There are two types of ways to implement data storage.
1. CC sector and receiving the deal and storing data which are not been verified by a notary (human) called the unverified deal.
2. Receiving the deal and storing data from clients who are verified by a notary (human) called the verified deal.

**Problem Statement**
Since the mainnet launch, the Filecoin global SPs who are the builders of the network and the beneficiaries of the block rewards have had a number of grievances with the Filecoin project itself, including but not limited to the listed below:
1. The reduction of block rewards due to the RBP being lower than the network baseline. 
2. The deterioration of the global economy and the broader bear market. SPs are not able to make ends meet due to the token (FIL) price drop and low ROI;
3. The constant abuse and junk data have caused much controversy in the Community;
4. Individuals or institutions (including SPs) leave the Filecoin community due to the continued decrease in block rewards due to the RBP reduction and stop sector renewals or dump hardware while spreading negative news.

**Proposal Design and Principles**
**Objectives**
One of Filecoin's missions is to provide useful storage capacity, and the SPs that support this capacity are unable to profit from block rewards and the secondary market (selling of FIL) to maintain the network. The continuous reduction of the RBP and the constant release of tokens from sector expiration will affect the security of the network and disrupt supply and demand in the secondary FIL market. Therefore, the objectives of this FIP are as below:
1. Stimulate the circulation of FIL through financial means;
2. Increase Filecoin TVL and RBP;
3. Increase the block reward for SPs;
4. Accelerate FIL deflation so the FIL token can better maintain its value.

**Principles**
1. Minimal implementation changes;
2. No change in project vision;
3. Technically feasible;
4. Secure and reliable;
5. Optimal timing.

**Proposal**
1. Increase CC sector quality multiplier from 1x to 5x；
2. Increase the quality multiplier for unverified deals from 1x to 5x；
3. Increase the quality multiplier for verified deals from 10x to 20x;
4. The last network upgrade in 2023 or the first network upgrade in 2024  (before April 2024, Bitcoin halving and the return of bull market)

**Advantages**
1. Reduced market liquidity and circulating supply of FIL
2. Create room and space for  FIL+ auditing to ensure that useful data is archived in the Filecoin network. The 20X rewards can be shared with the data owner to encourage them to use the Filecoin network.
3. SPs can choose unverified deals to increase 5X block rewards, reducing controversy and pressure to game the system for the FIL+ community

The DC SPs will benefit from the block rewards, while the high block rewards will also subsidize the data providers, giving them more opportunities to bring in more valuable data onboarding. Moreover, Fil+ abusers can now go for the 5x option, alleviating the pressure to abuse and allowing actual clients to enjoy the subsidy provided by Filecoin.

As an SP and FIL holder, everyone should care about locking or reducing the circulating supply because it can directly affect the value of the FIL. Here are some reasons:

1. **A slow increase in supply:** When a token's supply increases slowly, its value can be more stable and less volatile, like BTC or ETH. This can provide FIL holders with a better sense of confidence and predictability.

2. **Incentivizing mechanisms:** Locking or reducing circulating supply can create additional demand for FIL. For example, if FIL are locked in a smart contract/network for a certain period of time, this can reduce velocity and drive up the value of the remaining tokens.

3. **Stability:**  This can provide FIL holders with more confidence regarding the predictability and stability of their purchasing power.

4. **Long-term value:** Locking or reducing circulating supply can signal long-term confidence in the project and ecosystem. This can create a sense of trust among network participants and help to establish a more sustainable token economy over the long term.

5. **Kick-start a new phase:** QAP growth has been sluggish and RBP declining. This proposal can kick-start growth, setting up the network to take advantage of the next phase of the macro supercycle. 
Therefore, as an SP and FIL holder, I am very concerned about it. I believe that locking or reducing the circulating supply is an important factor in boosting the strength of the economy.

**Disadvantages**
1. Increased load factor for SPs;
2. SPs will not be able to expand their business in the coming year if they do not join the loan programme.

**Open Questions**
If the proposal passes,
1. What is the deflation rate of FIL?
2. How much can the network RBP and TVL increase?

---------------------------------------分割线----------------------------------------------------

Filecoin 提案讨论：提高Filecoin网络算力乘数的原因和建议
作者：1475Notary
类型：技术
类别：核心
2023-09-15

**前言**
本次FIP主要解释我们为什么要提高网络算力乘数。我们将讨论现有算力乘数的增加后的利弊及提高全网算力乘数的建议。

**背景**
使用Filecoin网络巨大存储空间（Proof of Space）来存储数据是项目的重要愿景之一。通过Fil+项目为Filecoin网络引入了一个许可层，通过算力乘数将原值算力（RBP）调整为原值算力的10倍（QAP），这意味着增加了9倍获得区块奖励的机会。算力乘数用于补贴参与Fil+计划的存储提供商，他们接收并存储经公证人（人类）验证过的数据从而获得10倍的挖矿算力， 作为回报存储提供商将为数据所有者提供稳定，长期，具有竞争力价格的存储服务。
实现数据存储的方式有两种类，
1.  CC数据/接收并存储未经公证人（人类）验证订单中的数据（unverified deal）；
2.  接受并存储经公证人（人类）验证订单中的数据(verified deal)。

**问题陈述**
从主网上线至今,全球存储提供商作为网络的构建者和区块奖励获得者对于Filecoin项目本身存在一些不满，包括但不限于以下几点：
1.  原值算力低于网络基线带来的区块奖励减少；
2.  全球经济恶化以及更广泛的熊市使SP无法维持生计，由于代币（FIL）价格下跌和低投资回报；
3. 不断的滥用和垃圾数据在社区引起了很大争议；
4. 个人或机构（包括 SP）因为RBP 减少导致区块奖励持续减少而离开 Filecoin 社区，他们停止扇区续期或抛售硬件，同时传播负面消息。

**提案设计目标及原则**
**目标**
Filecoin的使命之一是提供有用的存储能力，而支撑这一能力的存储提供商无法从区块奖励和二级市场（售卖FIL）中获利来维持网络，持续减少的原值算力和来自扇区到期释放的代币将极大的影响网络安全和破坏二级市场的供求关系，基于此，本FIP的目标如下：
1.	通过金融手段刺激代币流通；
2.	网络锁定总价值TVL的提高和原值算力RPBP的增加；
3.	提高存储提供商的区块奖励；
4.	加速 FIL 通缩，以便 FIL 代币能更好地保持其价值。

**原则**
1. 最小化实施变更
2. 项目愿景不改变
3. 技术上可行
4. 安全可靠
5. 最佳的时机

**提案**
1. 将CC扇区的乘数由1倍提高至5倍
2. 将未验证订单的算力乘数有1倍提高至5倍
3. 将验证订单的算力乘数由10倍提高至20倍
4. 2023年的最后一网络升级或2024 年的第一次网络升级（比特币减半和牛市回归之前，2024 年 4 月之前）

**优点**
1. 减少了 FIL 的市场流动性和流通供应；
2. 为 FIL+ 审计创造了空间，确保有用数据被归档在 Filecoin 网络中。20X 奖励可以与数据所有者共享，以鼓励他们使用 Filecoin 网络。
3. SP 可以选择未经验证的交易来增加 5X 区块奖励，减少争议和戏弄 FIL+ 系统的压力。

存储验证订单数据的SP将从区块奖励中受益，而高额的区块奖励也将补贴数据提供者，给他们带来更多机会使得更有价值的数据加入。此外，Fil+滥用者现在可以选择 5x 的选项，减轻滥用的压力，并让实际客户享受 Filecoin 提供的补贴。

作为存储供应商和 FIL 持有者，每个人都应该关注锁仓或减少循环供应，因为它可以直接影响 FIL 的价值。以下是其中的一些原因：
1. **控制供应量**： 当一个代币的供应量缓慢增加时，它的价值可以更加稳定，波动性更小，例如比特币（BTC）或以太坊（ETH）。这可以为 FIL 持有者提供更好的信心和可预测性。
2. **激励机制**： 锁定或减少循环供应可以创造对 FIL 的额外需求。例如，如果 FIL 在一个智能合约/网络中被锁定一段时间，这可以降低流通速度并推高剩余代币的价值。
3. **稳定性**： 这可以让 FIL 持有者对他们的购买力的可预测性和稳定性更有信心
4. **长期价值**： 锁定或减少流通供应可以表明代币背后的项目或团队的长期愿景。这可以在投资者之间建立一种信任感，有助于建立一个更具长期可持续性的代币经济。
5. **启动新阶段**：QAP 增长一直缓慢，而 RBP 不断下降。这个提案可以启动增长，使网络能够利用下一个宏观超级周期的阶段。

因此，作为一名存储提供者和 FIL 持有者，我非常关注它。我认为锁定或减少循环供应是促进经济增长的重要因素。

**缺点**
1. 提高了存储提供商的负债率；
2.  存储提供商不使用借贷协议，来年将无法扩大业务。

**开放式问题**
如果方案得以推行，
1. FIL的通缩率是多少？
2. 网络原值算力和TVL分别能增加多少？

**Comment by michael8908** [comment]
Has no any reference to the most import issue -- corruption
完全没有提及最重要的问题 -- 腐败问题

**Reply by 1475Notary** [reply]
Hi, @michael8908 Thank you for your reply, this FIP focuses on increasing the multiplier, if you have better ideas and suggestions on corruption, feel free to propose a new discussion, I'm sure we'd all like to participate. 
感谢你的回复，这个FIP主要讨论提高算力乘数的问题，如果您在腐败问题上有更好的想法和建议，欢迎提出一个新的讨论，我相信大家都愿意参与。

**Reply by michael8908** [reply]
@1475Notary 
corruption has a positive relation with the multiplier. The more multiplier, the more corruptive motivation. So it makes sense that the two should be considered synchronously. Although I have no a good idea on the corruption issue, to make decision under neglecting the possibility of corruption maye takes in trimendous prices to correct it in the future. Just a personally kind reminder.

算力乘数与腐败可能性是正向相关的，算力乘数越大也就意味着越有腐败动机，所以有理由认为两者应该同步考虑。虽然我个人并没有太好的方案，但是在忽略腐败可能性的前提下决策，也许未来纠正会付出极大的代价。个人的一个善意提醒。

**Reply by 1475Notary** [reply]
Thanks @michael8908  What we want to convey under this proposal is that the scrutiny of DataCap (DC) will be more rigorous than ever before, and the governance team also hopes to bring more data into the network through this proposal.
**"Furthermore, Fil+ abusers can now opt for a 5x option, alleviating the pressure of abuse."**

我们想表达的是在这个提案下，DC的审核将前所未有的严格，治理组也希望通过这个提案将更多的数据引入到网络中来，
**“此外，Fil+滥用者现在可以选择 5x 的选项，减轻滥用的压力”**

**Comment by Asher774** [comment]
Let the free market decide the most efficient way to decide the money. Let's see how many SPs would like to do it. Do you guys plan to vote on it?

**Reply by 1475Notary** [reply]
Thanks for your comments. @Asher774 

No, we don't plan to vote now, we want to hear the voice of the community no matter positive or negative.

Admittedly, in a bear market, we hope everyone comes together,  we are not expecting everyone to do it. Everyone has their own calculation on it. The death spiral of FIL is everyone's failure, and no one in this community wants to see that.

**Comment by flyworker** [comment]
```
Increase CC sector quality multiplier from 1x to 5x；
Increase the quality multiplier for unverified deals from 1x to 5x；
Increase the quality multiplier for verified deals from 10x to 20x;
```

those proposal just make the cost to product 1 Fil even cheap. anyone who buy filecoin for collateral will get 10% ~20% value in USD after the sectors expired.

It is seems help to do the TVL, but actually make Filecoin less value.

Cannot Support.

**Reply by 1475Notary** [reply]
Hi Charles, Thanks for your comment.

What I'm trying to point out is that you're using a dynamic behaviour to arrive at a static result. 
Would the situation you describe be improved if locking and releasing were done in parallel? 

I'm also very interested in hearing what good ideas and suggestions you, as a seasoned community member/ex-notary, have for all the SPs out there who are suffering. 

Thanks!

**Comment by Martin-0012** [comment]
和去年的提案很相似，大家都质押锁仓嘛没有新意！

**Reply by 1475Notary** [reply]
感谢你的回复！是的，本提案针对目前FIL流通大而做出的措施。解决问题不一定需要创新，除非在这痛苦的熊市中能有更好办法。我们也欢迎听到你的声音和建议。

**Reply by 1475Notary** [reply]
借币封装收益是最大化的，但得具备规划能力才能在高位挣到更多

**Comment by herony-fil** [comment]
The reward source for （verified or unvierified)deal should be from the 15% retrieval reward allocation, not the CC miner reward pool. All sectors should be same power rate and there should be no multipliers.

**Reply by 1475Notary** [reply]
@herony-fil Thanks for your comment. What I do know is that Saturn team(retrieval) will have its own token economy. 

The multiplier is a tool to help data onboarding, once the user habits are established, it can be canceled by another FIP.

Let's focus on this FIP, Do you have any suggestions on how to improve Filecoin in this bear market?

**Comment by Emmetttian** [comment]
As a community member what I suggest is that SP should approach the engagement in 3 layers: 

1.  The bottom-up analysis, looking back to their balance sheet, reduces their cost, layoffs if possible, to make sure you have the cash to survive during the bear market.

2. A strategic assessment of issues that can impact an SP across the business income segment, for example: find the client who would like to pay for the storage not only winning the block rewards. Using the levers (Defi protocol) to enlarge the capability of winning block rewards, etc.

3. Third is collaborative engagement, where SP gets support from other investors or data owners to slove ROI issues or storage demand, by joining E-FIL+ or FIL+.

**Reply by 1475Notary** [reply]
@Emmetttian I appreciate your time and consideration, and I'm really glad you're sharing your approach with the community, would you mind reaching out me on Slack by handle 1475Notary.

**Comment by 1475Notary** [comment]
Guys, you are also welcome to join the discussion on Slack,
**fil-Chinese**
https://filecoinproject.slack.com/archives/CR98WERRN/p1695224470108689

**fil-plus**
https://filecoinproject.slack.com/archives/C01DLAPKDGX/p1695211387588909

**Comment by xiaoliwe** [comment]
From the perspective of network growth, **I support this proposal;** 
perhaps it is a more appropriate plan at this stage.

**Reply by 1475Notary** [reply]
Thanks, @xiaoliwe , you can reach me via Slack for further discussion by handle  1475Notary

**Comment by pandacrypto** [comment]
After reading through the proposal, I do agree with @1475Notary proposal. 

The point you wrote locking or reducing the circulating supply because it can directly affect the value of the FIL, this is the most important point why we need this proposal to go forward, if there are changes and improvement required, we can take this opportunity to address it now, thus we can see the network growth in FVM.

**Reply by 1475Notary** [reply]
Yes 100% agree with you @pandacrypto , hope you share more to the SP and join us together.

**Comment by AlanGreaterheat** [comment]
Analyze this proposal from a SP perspective and support it @1475Notary . More urgent and important than DC abuse is that SP lives first.

**Reply by 1475Notary** [reply]
Thanks @AlanGreaterheat  I am sure you can involve more SP join it from DSPA

**Comment by SmallMiner** [comment]
We need to pull the network out of the vortex quickly. This proposal may be the only valid option at this time.

Willing to support the three multiplier changes.

**Reply by 1475Notary** [reply]
Thanks @SmallMiner  Please involve more small SP join my proposal.

**Comment by michael-421** [comment]
I do agree with this proposal. Any FIL lending project you would like to recommend? @1475Notary   Both Anchorage and Coinlist can not support us.

**Reply by 1475Notary** [reply]
Hi @michael-421 Thanks for your comment. You can try [Glif](www.glif.io) or [STFIL](www.stfil.io) for FIL lending

**Comment by Nur-Seng** [comment]
100% support, no pain no gain! But I suggest @1475Notary you ask Juan's team to join SPs for the deflation. Otherwise we make the price nice but PL gets the most out of it.

**Reply by 1475Notary** [reply]
Thanks @Nur-Seng , What do you mean by Juan's team? PL or FF?

**Comment by kkarrancsu** [comment]
Hi all,
CEL performed some analysis on this proposal, as well as some related variants. Please see the detailed report [here](https://hackmd.io/Lx6AKHmRSe2L5dDzW4zrTw?view), and a higher level blog post [here](https://medium.com/cryptoeconlab/cryptoeconomic-impacts-of-universal-multipliers-in-the-filecoin-economy-0b9bad76411a).

**Reply by 1475Notary** [reply]
Thank you @kkarrancsu  for your objective and rigorous analysis. I will share your insights with the community for further consideration.

**Comment by anorth** [comment]
Thanks for opening this discussion.

> As an SP and FIL holder, everyone should care about locking or reducing the circulating supply because it can directly affect the value of the FIL.

I think this point is well made, and seems to be becoming increasingly widely accepted among network participants today. While leasing or locking pledge collateral for new onboarding represents an immediate cost, resulting in a first desire to wish it were lower, one should also think about pledge requirements as a requirement on all the other SPs who might compete with you. Requiring future SPs to lock collateral protects an SP's investment, both in terms of maintaining a hurdle to competition, and stabilising the value of the future tokens to be earned as reward.

So let's say we can get some broad agreement on the desirability of increasing locking across the network. There are many different ways that this might be achieved, and I think it's well worth our time to consider multiple of them, rather than just the easy knobs to tweak. Some knob-tweaking ends up fairly ineffective because balancing mechanism come into play to push the aggregate result back to where it was. I think adjusting multipliers suffers from some of this, because the 30%-of-supply lock target reacts to reduce initial pledge requirements if the rate of pledging goes up.

Some directions that are worth consideration:
- Changing mandatory multipliers on new sectors (this proposal)
- Changing the 30%-of-circulating-supply lock target
    - This would have a much more durable effect than changing multipliers. It was rejected as part of FIP-0036, but might be more acceptable (1) stand-alone, and (2) as a gradual change.
- Allowing free choice of multipliers (e.g. FIP-0078 #844, or other variants)
    - This allows SPs to increase their pledge-to-hardware ratio without the frictions of FIL+ program. To the extent that the FIL+ program has succeeded in increasing QAP, this could achieve even more
- Pledge shortfall (#666)
    - SPs commit to locking future tokens they haven't earned yet, essentially incentivised reinvestment. Opposite to free multipliers, allows an SP to temporarily decrease their pledge-to-hardware ratio to onboard more hardware, but they must lock up the pledge eventually. Also burns some fees.
- Greater decoupling of pledge from storage, allowing and incentivising additional locking beyond that required for sector initial pledge

**Reply by 1475Notary** [reply]
Thank you @anorth for your support and suggestions. Let's unite as a community to protect the network during this downturn.

Now, let me share the viewpoints of those who agree with me from an SP's perspective according to your 5 points

 
1. I suggest implementing this FIP proposal. 

2. We think that raising the initial pledge may cause controversy among the community. FIP0036 didn't reach consensus before, and I believe it's challenging to reach a new consensus now, as people look towards the future.

3. Allowing the option for various multipliers would demand more development time and may not achieve a suitable balance between SP earnings and liquidity, creating difficulties in consensus formation during a bear market. This plan may cause SPs to be less concerned about bringing data onto the network, contradicting Filecoin's Master Plan of onboarding valuable data.

4.  there is little discussion and agreement on #666, and its development may take longer, making it challenging to form community consensus. 

5. While we believe this approach could work by spending another 6-9 months on development, it may divert the attention of SPs from the significant Filecoin project concept of data onboarding. If collateral and storage are separated, investors may focus more on FIL returns rather than onboarding valuable data.

You are welcome to make suggestions regarding this FIP. 

Thanks again.

**Reply by anorth** [reply]
> We think that raising the initial pledge may cause controversy among the community

I'm a bit confused by this objection. I interpreted your whole premise to be that you want to increase locking, most of which comes from initial pledge. Increasing multipliers will also increase per-sector initial pledge amounts (by a lot more!) - wouldn't that cause controversy? But unlike raising the supply target, the effect of higher multipliers won't stick because the mechanism will then decrease the pledge requirement and rate of locking back towards maintenance of 30%. It won't achieve the goals.

Perhaps you are proposing that multipliers are better because the QAP and share of rewards also increases, so notionally there's no immediate effect on ROI (and less controversy?). But this is failing to take into account the evolution over time. If all other SPs gain the same multipliers, then the network QAP will increase faster, your share of power will decrease more rapidly, and you'll have ended up paying a 2-5x pledge (vs current) but end up with much less than 2-5x share of rewards. It will initially appear like no change in ROI, but actually be worse over time, and SPs will be unable to make ends meet again. New onboarding would rapidly take share of QAP from existing storage though, squeezing its ROI harder; we can debate whether new onboarding getting an advantage to existing commitments is good or not, though.

On the other hand, despite initially higher pledges per sector, multipliers cannot sustainably increase the amount of locked tokens. The 30% target will react to reduce per sector pledge requirements. Very roughly, if the average multiplier is doubled and sector onboarding constant, the average pledge-per-QAP requirement will tend to halve, to result in the same rate of locking as before. It will also react if onboarding increases: more sectors -> lower pledge per sector.

Increasing multipliers cannot sustainably achieve the goal of "Increase Filecoin TVL and RBP". It can have a short-term effect (mainly at the expense of existing QAP), but the system will react to produce the same sector ROI and network TVL as before. It cannot accelerate FIL deflation - the amount locked and rate of locking will tend to the same 30%.

Increasing the 30% pledge target _can_ sustainably increase TVL. It's the most direct way to do so without a much larger change (option 5). Any other attempt to increase locking is thwarted by this target reacting in the opposite direction.

It's hard to increase RBP while also increasing costs such as pledge. For SP business stability we could either consider trying to increase rewards or decrease inflation. I think that decreasing inflation is much more likely to produce good outcomes (measured in fiat). Any increase in rewards is somewhat self-defeating as it increases inflation, decreasing the value of future rewards.

(The one sustainable change in the initial proposal is to reduce the advantage of FIL+ over CC to 4x, rather than 10x. If this increases the share of rewards available to CC sectors, it could prompt increasing the RBP:QAP ratio, which might hold block rewards marginally higher than otherwise. But we could achieve the same by just reducing the FIL+ multiplier, or allowing free multipliers. Yes they all affect incentives to onboard FIL+ data.)

**Reply by 1475Notary** [reply]
Thank you for sharing, @anorth ! As a community member and an early SP participant, I believe my perspective represents the views of a significant portion of the community:

1. The concept of increase the initial pledge mentioned in FIP-0036 has not yet formed a consensus within the community.

2. Increasing the initial pledge adds more risk for SPs and investors due to price volatility.

3. Increasing the initial pledge does not necessarily lead to higher returns.

In summary, while raising the initial pledge could indeed enhance deflation, it may not receive widespread community support considering human nature. 

An ideal proposal would involve both an increase in the initial pledge and the multiplier. However, do you think such a proposal could gain consensus within the community?

Furthermore, although this proposal increases the total pledge amount, it also increase the opportunity for SPs to get the block rewards. From a human perspective, this seems more logical and is something that both SPs and investors are willing to participate in.

**Comment by BSPN-TV** [comment]
1. I think the computing power of the CC sector should be 1 times the pledge and 5 times the computing power. DC sector 20 times pledge 20 calculation example. Maybe the value I gave is not very accurate and a balanced value should be obtained. Reason: Because the cost of CC miners lies in hardware, hosting, etc. Secondly, there is no high-value data in the CC sector, so there is no need for high mortgage rates. Then the DC sector stores valuable data, which requires high collateralization.
2. The minimum lock-up period of the CC sector should be 90 days, because no valuable data is stored, so there should be no long-term constraints.
The minimum period of DC should be more than one year, because the effective data is probably stored for more than one year.

**Reply by 1475Notary** [reply]
Hello, @BSPN-TV, thank you for your speech on SPACE that day. 

As an SP, I can truly appreciate your accounting logic, and I also agree with your proposal regarding the current decrease in the network's raw power (1x pledge, 5x QAP would effectively drive SP's activity on CC sectors).

However, such a proposal should be reconsidered in terms of fairness across the entire network.

1. The cost difference in hardware and rack fee in data center between CC miners and DC miners has not reached a 1:10 ratio.
2. CC miners provide " proof of storage," and they should have the same sector commitment period as DC miners; otherwise, the stability of the network's storage space cannot be sustained.

Just think, if the network's storage space can only last for 90 days, how can clients obtain 180 days' worth of storage deal.

**Comment by BSPN-TV** [comment]
1、我认为CC扇区的算力应该是1倍质押5倍算力。DC扇区20倍质押20算例。或许我给的数值不是很准确，应该取到一个平衡值。理由：因为CC矿工的成本在硬件、托管等方面。其次CC扇区里面没有高价值的数据，所以不需要高倍抵押。然后DC扇区存储的是有价值的数据，从而需要高倍质押。
2、CC扇区最低锁仓周期应该是90天，因为没有存储有价值数据，从而不应该有长周期约束。
DC最低周期应该是1年以上，因为有效数据大概都是一年以上的存储周期。

**Reply by 1475Notary** [reply]
@BSPN-TV 你好，感谢你在SPACE当天的发言。作为SP来讲非常理解你的核算逻辑，也赞同你对目前全网原值算力下降而提出的建议。（1倍质押5倍算力能有效提高CC扇区的封装积极性）

但这样的建议在全网公平性上还需重新思量。
1 CC矿工和DC矿工的硬件和托管成本区别并没有达到1:10，
2 CC矿工提供的是“存储空间证明” 它应该具备和DC矿工同样的扇区承诺周期，否则全网存储空间的稳定性就不能持续。

试想网络的存储空间只能持续90天，怎么能获取180天的存储订单？

**Reply by lyjmry** [reply]
拿到DC订单需要重新计算扇区周期

**Comment by The-Wayvy** [comment]
@1475Notary 

Why not raise CC to 2.5x and leave DC unchanged?

**Reply by 1475Notary** [reply]
I argee with you in part. @The-Wayvy 

If we are looking forward a little bit for the main purpose of this FIP

1. Reduced market liquidity and circulating supply of FIL.              |     ✓
2. Create room and space for FIL+ auditing.                                   |     X
3. Reducing controversy and pressure  for the FIL+ community.    |     X

Your suggestion only keep the first one. Please join me to make the 2& 3 smoother for the community if you have better idea.

Thank you.

**Reply by The-Wayvy** [reply]
@1475Notary 

Raising CC from 1 to 2.5 while keeping DC the same does address points 2 and 3 because it decreases miner demand for DC. 

IMO we should keep raising CC by 2.5 every few months until both CC and DC are 10x.

**Comment by The-Wayvy** [comment]
@1475Notary 

Under your proposal, can _existing_ sectors upgrade to 5x / 20x?

**Reply by 1475Notary** [reply]
Hi @The-Wayvy 

To be honest, from an SP perspective, I'm personally willing to do that. You can quickly get block rewards by increasing your pledge, and then sell them when you leave. :-) 

However, from a PL's perspective, it seems a bit unfair. Network security requires a steady and healthy growth of both power and pledge. If there's a sudden increase in power and pledges within a short period due to this FIP, it can lead to a significant drop in power and pledges after that period. This is very dangerous. Please think of the LUNA crash; I don't think anyone wants the FIL to experience something like that.

At the same time, this suggestion doesn't provide a positive direction for real data onboarding and doesn't help steer the community in the right direction.

**Reply by The-Wayvy** [reply]
@1475Notary 

If existing CC sectors were not allowed to upgrade, this proposal would be **_extremely_** unfair to existing CC miners.
 
I would expect mass termination of 1x CC sectors.

**Comment by nj-steve** [comment]
I think the proposal that can retain CC's power is reasonable, and it can balance the interests of some participants. As for the computing power coefficient, we need to discuss it.
